template/bash_profile.template.bash

```diff
@@ -34,6 +34,9 @@ export IRC_CLIENT='irssi'
 # Set this to the command you use for todo.txt-cli
 export TODO="t"
 
+# Set this to the location of your work or project folders
+#BASH_IT_PROJECT_PATHS="${HOME}/Projects:/Volumes/work/src"
```

Comment:
you forgot the `export` 😄

Reply:
Nope! Almost none of these should be exported, but I didn't fix the others here. 😃

Reply:
you are absolutely correct